src/pandas_profiling/profile_report.py

```diff
@@ -50,6 +50,7 @@ def __init__(
             sample: optional dict(name="Sample title", caption="Caption", data=pd.DataFrame())
             **kwargs: other arguments, for valid arguments, check the default configuration file.
         """
+        config.clear()  # to reset (previous) config.
```

Comment:
Good addition. We should move away from global config someday...

Reply:
There's only one way to eat an elephant :).